python/py-pyaudio/Portfile

```diff
@@ -8,7 +8,7 @@ set my_name             PyAudio
 version                 0.2.11
 categories-append       audio
 platforms               darwin
-maintainers             NLnetLabs.nl:jaap
+maintainers             {@Jakker NLnetLabs.nl:jaap}
```

Comment:
Would it also be okay to add `openmaintainer` at the end?

Reply:
Oops, I actually originally planned to do that and then apparently forgot. Need a new commit for that?

Reply:
An amended commit is fine for such trivial changes.

Reply:
Did an amended commit. Hope I did it right (I'm a git newbie).

Reply:
Well, looks like the change is not in. You can check overall changes at https://github.com/macports/macports-ports/pull/2486/files

Reply:
Apologies for the noise, but the change is now in.